sources/install/package_network.sh

```diff
@@ -9,7 +9,9 @@ function install_network_apt_tools() {
     export DEBIAN_FRONTEND=noninteractive
     fapt wireshark tshark hping3 masscan netdiscover tcpdump iptables traceroute dns2tcp freerdp2-x11 \
     rdesktop xtightvncviewer ssh-audit hydra mariadb-client redis-tools
-    fapt remmina remmina-plugin-rdp remmina-plugin-secret remmina-plugin-spice
+    fapt remmina remmina-plugin-rdp remmina-plugin-secret
+    # remmina-plugin-spice need build ?
```

Comment:
`remmina-plugin-spice` no longer exists under debian12 ? Is it needed for the build ?

Reply:
yes it is no longer in the repository but I was not able to check if the plugins were not integrated into the main package